### PR TITLE
Support v0.73.0 swap claims

### PR DESCRIPTION
**Pull request by grod220**
Closes https://github.com/penumbra-zone/web/issues/1004

### CODE REVIEW

File: apps/extension/.env.testnet [hunk 1]
```diff
@@ -1,4 +1,4 @@
 CHAIN_ID=penumbra-testnet-deimos-6
-IDB_VERSION=34
+IDB_VERSION=35
```
Comment (grod220) [inline apps/extension/.env.testnet:2]: So we can get those old swaps stored with sct positions

File: packages/transport-dom/src/create.ts [hunk 2]
```diff
@@ -184,7 +185,7 @@ export const createChannelTransport = ({
             Code.OutOfRange,
           );
       } else {
-        const stream = ReadableStream.from(input).pipeThrough(
+        const stream: ReadableStream<JsonValue> = ReadableStream.from(input).pipeThrough(
```
Comment (grod220) [inline packages/transport-dom/src/create.ts:188]: New lint error from protobuf pkg update

File: packages/wasm/crate/src/tree.rs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+#[wasm_bindgen]
+pub fn sct_position(block_height: u64, epoch_bytes: &[u8]) -> WasmResult<u64> {
+    let epoch = Epoch::decode(epoch_bytes)?;
+    let epoch_index = u16::try_from(epoch.index)?;
+    // The block index is determined by looking at how many blocks have elapsed since
+    // the start of the epoch.
+    let block_index = u16::try_from(block_height - epoch.start_height)?;
+
+    let position = Position::from((epoch_index, block_index, 0u16));
+    Ok(position.into())
```
Comment (Valentine1898) [inline packages/wasm/crate/src/tree.rs:17]: Seems like we could have done it without wasm?

**Review by Valentine1898 — APPROVED** [review]
LGTM!

File: packages/wasm/crate/src/tree.rs (hunk 3 above)
Comment (grod220) [reply packages/wasm/crate/src/tree.rs:17]: The tree position is serialized using bitwise operators: https://github.com/penumbra-zone/penumbra/blob/8d4dc0330a8bf86a4334987a02f0fa04ac40baa0/crates/crypto/tct/src/index.rs#L228-L238.

I thought about implementing a version of this in typescript, but I figured 1) it's easier to use what they created and 2) we'll be attempting to migrate this to wasm soon anyway.

Comment (Valentine1898) [reply packages/wasm/crate/src/tree.rs:17]: Makes sense

**Pull request merged**